src/name_bazaar/server/db_sync.cljs

```diff
@@ -39,7 +39,7 @@
                 (= (second (<! ens-owner-ch))                 ;; For other names just basic ENS ownership check
                    offering-address))))
         (catch :default e
-          (logging/error {:error e}))))
+          (logging/error {:error (d0x-shared-utils/jsobj->clj e)}))))
```

Comment:
if you use this function so many times in a file please use `:refer`. Apply to other files as well